bindingcollectionadapter/src/main/java/me/tatarka/bindingcollectionadapter2/itembindings/OnItemBindClass.java

```diff
@@ -44,6 +74,26 @@ public int itemTypeCount() {
 
     @Override
     public void onItemBind(ItemBinding itemBinding, int position, T item) {
+        itemBinding.clearExtra();
```

Comment:
This will clear extras set with `bindExta` which will break on-off usages like the one in the README.